portal-api/src/external-api/bank-holidays.js

```diff
@@ -11,7 +11,7 @@ const headers = {
 
 /**
  * Resolves to the response of `GET /bank-holidays` from external-api.
- * @returns {Promise<import('axios').AxiosResponse>}
+ * @returns {object}
```

Comment:
Seeing as the type of bank holidays is fixed in external API, can be add a JSDoc typedef for it to use here?